includes/multi-currency/Compatibility/BaseCompatibility.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Class BaseCompatibility
+ *
+ * @package WCPay\MultiCurrency\Compatibility
+ */
+
+namespace WCPay\MultiCurrency\Compatibility;
+
+use WCPay\MultiCurrency\MultiCurrency;
+use WCPay\MultiCurrency\Utils;
+
+/**
+ * Class that sets up base options for compatibility classes.
+ */
+class BaseCompatibility {
```

Comment:
I think it'll be better to mark this class as abstract, to prevent it from getting instantiated. What do you think?

Reply:
Thanks for the suggestion, updated in https://github.com/Automattic/woocommerce-payments/pull/3259/commits/0f0dff55744bcb0d3c04dea2d1a6b7da5e6147e7